package.json

```diff
@@ -25,22 +27,25 @@
     "prepare": "patch-package",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish-assets:local": "yarn assets && yarn publish-assets",
     "publish-assets": "node scripts/uploadToS3.js",
+    "publish-assets:local": "yarn clean && yarn build:assets && yarn publish-assets",
     "relay": "relay-compiler",
     "sentry": "scripts/sentry.sh",
-    "start:dev": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src",
-    "start:prod:local": "NODE_ENV=production node --inspect --max_old_space_size=3072 -r dotenv/config ./server.dist.js",
-    "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
+    "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
+    "start:dev:server": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src/index.dev.js",
+    "start:dev:relay": "NODE_ENV=development yarn relay --watch",
```

Comment:
We already have a `yarn relay` command, and yarn start runs `yarn relay --watch`. Whats the thinking behind this addition?

Reply:
I'll respond to all of the package.json comments here, it's hard to follow your thoughts as they are scattered throughout your review.

In short, yes development for devs hasn't changed in any significant way. I expect it to continue evolving over time and ensuring that existing commands continued to work was the easiest change to make without causing immediate and significant workflow issues.

I personally feel like clearly documenting the commands that are used for force in the `package.json` is preferable for several reasons.

1. Maintaining bash scripts is tedious, the shell usage is inconsistent, error handling is inconsistent, they only execute within a yarn/npm subshell, and is not cross-platform. Both npm and yarn have a minimal shell built in that attempts to solve some of the cross-platform issues.
2. Whenever you want to know a current command does, you incur a side trip penalty because you need to locate and process an additional file to find out what is happening. 
3. Take a look at the current `yarn start` script. Can you confidently tell me which command it will execute? I cannot without inspecting the current shell environment.
4. There are several instances where our code changes paths internally and relies on the current `NODE_ENV` to execute as expected. These changes clearly document that.
5. This is what the `package.json` is typically used for.

So yes there are more commands, they are each complete, and a developer can at a glance tell what that command is. Over time I would expect these "new" commands to slowly replace their counterparts in the scripts folder removing the appearance of duplication.

Reply:
Makes sense 👍 Can we remove the yarn start relay command in favor of existing relay command which the team is already familiar with?